docs/version-history/v3.0.md

```diff
@@ -0,0 +1,146 @@
+- HTML special character decoding does not preserve null input ([PR #1170](https://github.com/lk-arpg/lorekeeper/pull/1170))
+- Design update fullsize image saving does not take format from image if relevant ([PR #1169](https://github.com/lk-arpg/lorekeeper/pull/1169))
+- Gallery submission log errors if currency form sections don't match configured form ([PR #1166](https://github.com/lk-arpg/lorekeeper/pull/1166))
+- Gallery submissions do not list prompt submissions with different protocol (www vs without) ([PR #1173](https://github.com/lk-arpg/lorekeeper/pull/1173))
+- Rejected gallery submissions can be edited ([PR #1176](https://github.com/lk-arpg/lorekeeper/pull/1176))
+- Character design update thumbnails lack an extension with certain configurations ([PR #1181](https://github.com/lk-arpg/lorekeeper/pull/1181))
+- shorter masterlist image dimension target config setting does not work ([PR #1182](https://github.com/lk-arpg/lorekeeper/pull/1182))
+- Error viewing prompts ([PR #1185](https://github.com/lk-arpg/lorekeeper/pull/1185))
+- Collaborator approval scope does not filter gallery submissions with non-approving collaborators ([PR #1193](https://github.com/lk-arpg/lorekeeper/pull/1193))
+- Issues using WYSIWYG editor dialogs when replying to or editing comments ([PR #1216](https://github.com/lk-arpg/lorekeeper/pull/1216))
+- Some admin user editing breadcrumbs and page titles are incorrect or unclear ([PR #1233](https://github.com/lk-arpg/lorekeeper/pull/1233))
+- Character admin transfer notifications do not correctly link to transferred MYO slots ([PR #1246](https://github.com/lk-arpg/lorekeeper/pull/1246))
+- World rarity entry admin edit component display is inconsistent ([PR #1257](https://github.com/lk-arpg/lorekeeper/pull/1257))
+- Permissioned users cannot submit to "closed" galleries ([PR #1256](https://github.com/lk-arpg/lorekeeper/pull/1256))
+- Rarity entry edit information is missing on admin create/edit page ([PR #1269](https://github.com/lk-arpg/lorekeeper/pull/1269))
+- Issue with alias login if user alias snowflakes match ([PR #1252](https://github.com/lk-arpg/lorekeeper/pull/1252)); backported in [Commit `b0fd238`](https://github.com/lk-arpg/lorekeeper/commit/b0fd238122e37b50e9d3b06cb4e8a7ffc2914679)
+- Shop name uniqueness is not checked against existing shops on creation ([PR #1278](https://github.com/lk-arpg/lorekeeper/pull/1278))
+- Parsed rank descriptions are not set on initial setup ([PR #1280](https://github.com/lk-arpg/lorekeeper/pull/1280))
+- Notifications aren't sent to owners of characters included in gallery submissions ([PR #1281](https://github.com/lk-arpg/lorekeeper/pull/1281))
+- Custom CSS file is not reloaded when modified ([PR #1287](https://github.com/lk-arpg/lorekeeper/pull/1287))
+
+[Release notes](https://github.com/lk-arpg/lorekeeper/tree/release/v3.0.0)
+
+## Core Extensions Added
```

Comment:
No features were added after v3.0 entered feature freeze, so this can't be correct.

Reply:
That is a _very_ good point. And yet:
This is from the difference between the [v3.0.0-pre1 tag](https://github.com/lk-arpg/lorekeeper/blob/v3.0.0-pre1/resources/views/pages/credits.blade.php) and the [release/v3.0.0 branch](https://github.com/lk-arpg/lorekeeper/blob/release/v3.0.0/resources/views/pages/credits.blade.php).

I'll have to look into whether or not these were added early and only got credits later or.. if they really were added later.